sql/core/src/test/resources/sql-tests/inputs/interval.sql

```diff
@@ -220,3 +220,9 @@ select a - b from values (interval '-2147483648 months', interval '2147483647 mo
 select b + interval '1 month' from values (interval '-2147483648 months', interval '2147483647 months') t(a, b);
 select a * 1.1 from values (interval '-2147483648 months', interval '2147483647 months') t(a, b);
 select a / 0.5 from values (interval '-2147483648 months', interval '2147483647 months') t(a, b);
+
+-- interval support for csv and json functions
+SELECT from_csv('1, 1 day', 'a INT, b interval');
+SELECT to_csv(named_struct('a', interval 32 month, 'b', interval 70 minute));
+SELECT from_json('{"a":"1 days"}', 'a interval');
+SELECT to_json(map('a', interval 25 month 100 day 130 minute));
```

Comment:
@yaooqinn Could you add round-trip test cases?

- from_csv(to_csv(....))
- to_csv(from_csv(...))
- from_json(to_json(....))
- to_json(from_json(....))
-

Reply:
ok, I'll make a followup.